docs/how-to/build-wallets-signing-tools-api-clients.md

```diff
@@ -9,19 +9,21 @@ description: Additional building blocks for Filecoin applications
 
 To simplify or automate how users make payments, you can integrate one or more of these wallets directly into your application or service:
 
-**_TODO: Add wallet options and descriptions when ready_**
+- [Ledger integration library](https://github.com/Zondax/ledger-filecoin/) - for developers of wallets or applications wishing to integrate with the Ledger hardware device
+- [Glif web wallet](https://github.com/openworklabs/filecoin-web-wallet) (in progress) - web wallet built with Next.js that supports sending & receiving FIL from a Ledger device
+- Mobile and other wallet integrations (to come)
 
 
 ## Signing Libaries
 
 Signing libraries can be used to sign transactions without requiring a dedicated node. This allows you to create signed transactions in an application that is running offline, on a mobile device, or anywhere. The signed transaction can then be submitted to any Filecoin API endpoint.
 
-See (**_TODO add link when ready_**) for a list of available signing libraries.
-
+- [Filecoin Signing Tools](https://github.com/zondax/filecoin-rs) - a Rust/WASM/JSONRPC service for sending signed messages to the network
 
 ## API Clients
```

Comment:
Just confirming you intended to leave this as a level 2 header and have it surfaced in the nav?

Reply:
Yes for here - thanks!